src/amo/components/Addon/index.js

```diff
@@ -181,9 +182,13 @@ export class AddonBase extends React.Component {
     const authorList = addon.authors.map(
       (author) => `<a href="${author.url}">${author.name}</a>`);
     const description = addon.description ? addon.description : addon.summary;
+    // Themes lack a summary so we do the inverse :-/
+    // TODO: We should file an API bug about this...
+    const summary = addon.summary ? addon.summary : addon.description;
+
     const descriptionSanitized = sanitizeHTML(
       nl2br(description), allowedDescriptionTags);
-    const summarySanitized = sanitizeHTML(addon.summary, ['a']);
```

Comment:
The thought I was having is that it might make it easier to have a future lint rule that all content passed as HTML via dangerouslySetInnerHTML. should be wrapped in a sanitization function.